README.md

```diff
@@ -30,166 +32,32 @@
 
 </p>
 
-## **Overview**
-
-*earthaccess* is a **python library to search, download or stream NASA Earth science data** with just a few lines of code.
-
-
-In the age of cloud computing, the power of open science only reaches its full potential if we have easy-to-use workflows that facilitate research in an inclusive, efficient and reproducible way. Unfortunately —as it stands today— scientists and students alike face a steep learning curve adapting to systems that have grown too complex and end up spending more time on the technicalities of the tools, cloud and NASA APIs than focusing on their important science.
-
-During several workshops organized by [NASA Openscapes](https://nasa-openscapes.github.io/events.html), the need to provide easy-to-use tools to our users became evident. Open science is a collaborative effort; it involves people from different technical backgrounds, and the data analysis to solve the pressing problems we face cannot be limited by the complexity of the underlying systems. Therefore, providing easy access to NASA Earthdata regardless of the data storage location (hosted within or outside of the cloud) is the main motivation behind this Python library.
-
-## **Installing earthaccess**
-
-You will need Python 3.8 or higher installed.
-
-Install the latest release using conda
 
-```bash
-conda install -c conda-forge earthaccess
-```
+`earthaccess` is a python library to **search for**, and **download** or **stream** NASA Earth science data with just a few lines of code.
 
-Using Pip
-
-```bash
-pip install earthaccess
-```
+Visit [our documentation](https://earthaccess.readthedocs.io/en/latest) to learn more!
```

Comment:
Just a note that https://earthaccess.readthedocs.io/en/latest will, by default, show a banner warning users to try the stable version instead. @betolink updated the warning per guidance in https://docs.readthedocs.io/en/stable/versions.html#version-warning but we need to trigger a build to double-check that this is indeed removed.

Reply:
latest is also the default. If you visit https://earthaccess.readthedocs.io, it will redirect.